impeller/entity/contents/filters/filter_contents.h

```diff
@@ -15,6 +15,15 @@
 #include "impeller/renderer/formats.h"
 
 namespace impeller {
+using ImageFilterProc = std::function<std::shared_ptr<FilterContents>(
+    FilterInput::Ref,
+    const Matrix& effect_transform)>;
+using ColorFilterProc =
+    std::function<std::shared_ptr<FilterContents>(FilterInput::Ref)>;
+using MaskFilterProc = std::function<std::shared_ptr<FilterContents>(
+    FilterInput::Ref,
+    bool is_solid_color,
+    const Matrix& effect_transform)>;
```

Comment:
These type aliases are closely tied to the Paint state -- I'd rather we just make new ones if convenient elsewhere rather than moving them.

Reply:
Done.